x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/transport/CrossClusterAccessSecurityExtension.java

```diff
@@ -51,7 +53,7 @@ private CrossClusterAccessSecurityExtension(Components components) {
             components.authorizationService(),
             components.securityContext(),
             this.authenticationService,
-            this.crossClusterApiKeySignatureManager,
+            crossClusterApiKeySignatureManager,
```

Comment:
I pass the full manager here and only the verifier above because the manager will issue a new signer per remote cluster while we only have a single verifier per cluster.